src/obfl-specification.html

```diff
@@ -1320,6 +1320,29 @@ <h4 id="L836" class="include">page-number</h4>
 </dl>
 </div>
 
+<h4 id="L840" class="include">metadata</h4>
+
+<ins datetime="20200429T15:00:00">The metadata attribute can be added to any block in order
+  to send information through the dotify framework from obfl to pef.</ins>
```

Comment:
Several remarks here.
1. Apparently you have changed the PEF specification. Please make sure that this change is reflected in that specification. See http://pef-format.org/?page_id=7 (is there a "mtmse" copy of this?)
2. This specification already has a section about metadata. See http://braillespecs.github.io/pef/pef-specification.html#Error1. Please make sure that your change to the PEF format is reflected here.
3. I don't know which organisations are responsible for the PEF format. Have you contacted those organisations?

Reply:
@PaulRambags See https://mtmse.github.io/pef/pef-specification.html#Using

Reply:
@kalaspuffar Have you updated https://mtmse.github.io/pef/pef-specification.html?

Reply:
No, we don't change the pef specification.